packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -107,4 +112,121 @@ storiesOf('ListView', module)
     <ListView width="300px" height="300px" isLoading>
       {[]}
     </ListView>
-  ));
+  ))
+  .add('density: compact', () => (
+    <ListView width="250px" density="compact">
+      <Item textValue="row1">row 1</Item>
+      <Item textValue="row2">row 2</Item>
+      <Item textValue="row3">row 3</Item>
+    </ListView>
+  ))
+  .add('density: spacious', () => (
+    <ListView width="250px" density="spacious">
+      <Item textValue="row1">row 1</Item>
+      <Item textValue="row2">row 2</Item>
+      <Item textValue="row3">row 3</Item>
+    </ListView>
+  ))
+  .add('selection: none', () => (
+    <Example selectionMode="none" />
+  ))
+  .add('selection: single, checkbox', () => (
+    <Example selectionMode="single" />
+  ))
+  .add('selection: single, checkbox, disabled', () => (
+    <Example selectionMode="single" disabledKeys={['row1']} />
+  ))
+  .add('selection: multiple, checkbox', () => (
+    <Example selectionMode="multiple" />
+  ))
+  .add('parent link example', () => (
+    <Example2 selectionMode="multiple" />
+  ))
+  .add('actions: ActionButton', () =>
+    renderActionsExample(props => <ActionButton {...props}><Copy /></ActionButton>))
+  .add('actions: ActionGroup', () =>
+    renderActionsExample(props => (
+      <ActionGroup {...props}>
+        <Item key="add"><Add /></Item>
+        <Item key="delete"><Delete /></Item>
+      </ActionGroup>
+    )))
+  .add('actions: ActionMenu', () =>
+    renderActionsExample(props => (
+      <ActionMenu {...props}>
```

Comment:
Something is messing with the css in these menu items... Maybe a slot gone awry?